lib/set-delegates.js

```diff
@@ -206,6 +227,24 @@ module.exports = function(browserWindow, panel, webview, options) {
     })
   }
 
+  var themeObserver = ThemeObserverClass.new({
+    utils: {
+      executeJavaScript: browserWindow.webContents.executeJavaScript.bind(
+        browserWindow.webContents
+      ),
+    },
+  })
+
+  NSApplication.sharedApplication().addObserver_forKeyPath_options_context(
```

Comment:
in our implementation, I created `register` and `deregister`  methods on the `ThemeObserverClass` to tie up these `addObserver` and `removeObserver` calls - feels neater for if the keypath was to change or something